wbooks/src/app/screens/Library/index.tsx

```diff
@@ -1,19 +1,32 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { View, FlatList, ListRenderItem } from 'react-native';
-import { BOOKS_MOCK } from '@constants/mockBooks';
 import BookData from '@screens/Library/components/BookData';
 import Book from '@interfaces/book';
+import { useDispatch, useSelector } from 'react-redux';
+import { getBooks } from '@redux/book/actions';
 
 import styles from './styles';
 
+interface RootState {
+  books: Book[];
+}
+
 const Library = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
+
+  const books = useSelector((state: RootState) => state.books);
```

Comment:
when you modify `store.ts` the books will be in `state.books.books`

Reply:
Ok, fixed